lib/src/access.ts

```diff
@@ -29,59 +18,18 @@ export type RewrapResponse = {
  */
 export async function fetchWrappedKey(
   url: string,
-  requestBody: RewrapRequest,
-  authProvider: AuthProvider,
-  clientVersion: string
+  signedRequestToken: string,
+  authProvider: AuthProvider
 ): Promise<RewrapResponse> {
-  const req = await authProvider.withCreds({
-    url,
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(requestBody),
-  });
-
-  let response: Response;
-
+  const platformUrl = getHostFromEndpoint(url);
+  const platform = new PlatformClient({ authProvider, platformUrl });
   try {
-    response = await fetch(req.url, {
-      method: req.method,
-      mode: 'cors', // no-cors, *cors, same-origin
-      cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
-      credentials: 'same-origin', // include, *same-origin, omit
-      headers: req.headers,
-      redirect: 'follow', // manual, *follow, error
-      referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
-      body: req.body as BodyInit,
+    return await platform.v1.access.rewrap({
+      signedRequestToken,
     });
   } catch (e) {
-    throw new NetworkError(`unable to fetch wrapped key from [${url}]`, e);
+    throw new NetworkError(`[${platformUrl}] [Rewrap] ${extractRpcErrorMessage(e)}`);
```

Comment:
I've simplified this different instance of error thrown. The `error.message` contains the [RPC error codes](https://grpc.io/docs/guides/status-codes/) and in some cases the HTTP status code as well